rules/require-task-name-suffix.js

```diff
@@ -16,7 +16,22 @@ module.exports = {
           });
         }
       },
+      MethodDefinition(node) {
+        let { key, value, decorators } = node;
+        if (key.type !== 'Identifier' || key.name.endsWith('Task')) return;
+        if (!decorators) return;
+        if (!value) return;
+        if (!value.generator) return;
 
+        for (let decorator of node.decorators) {
+          if (decorator.expression.type === 'Identifier' && decorator.expression.name === 'task') {
```

Comment:
Should we also support the other kinds of decorators that now exist? I.e. `dropTask`, `restartableTask` etc.

For convenience a list: http://ember-concurrency.com/docs/task-decorators

Reply:
Ah good point, will add them now

Reply:
I've added a test case and checks for the other task decorators